deploy/crds/planetscale_v2_vitessshard_crd.yaml

```diff
@@ -679,6 +679,34 @@ spec:
               items:
                 type: string
               type: array
+            conditions:
+              additionalProperties:
+                properties:
```

Comment:
The `additionalProperties` -> `properties` tree confuses me a little, it feels like `additionalProperties` should be renamed or removed (I assume it serves a purpose though)

Reply:
This is auto-generated :-P